chunky/src/java/se/llbit/chunky/renderer/RenderManager.java

```diff
@@ -138,11 +138,15 @@ public RenderManager(RenderContext context, boolean headless) {
             finalizeAllFrames = scene.shouldFinalizeBuffer();
             updateRenderState(scene);
             if (reason == ResetReason.SCENE_LOADED) {
+              // Make sure frame is finalized
+              bufferedScene.postProcessFrame(renderTask);
```

Comment:
Why use the `renderTask`? If we use a new task tracker, the new `ostProcessFrame` overload wouldn't be needed. "Nested" tasks should work fine.